src/griffe/dataclasses.py

```diff
@@ -193,6 +195,12 @@ def __init__(
         """The parameter kind."""
         self.default: str | Expr | None = default
         """The parameter default value."""
+        self.docstring: Docstring | None = docstring
+        """The parameter docstring."""
+        # The parent function is set in Function.__init__ when the
+        # parameter(s) are assigned to a function
+        self.parent: Function | None = None
```

Comment:
Let's call it `function`.

```suggestion
        # The parent function is set in `Function.__init__`,
        # when the parameters are assigned to the function.
        self.function: Function | None = None
```